tests/Doctrine/Migrations/Tests/Configuration/ConfigurationTest.php

```diff
@@ -53,7 +54,18 @@ public function testNoNamespaceConfigured() : void
         $this->expectExceptionMessage('There are no namespaces configured.');
 
         $config = new Configuration();
-        $config->validate();
+        $config->freeze();
+    }
+
+    public function testFreezeConfiguration() : void
+    {
+        $this->expectException(FrozenConfiguration::class);
+        $this->expectExceptionMessage('The configuration is frozen and can not be edited anymore.');
```

Comment:
I'd move these 2 lines just before the call to `setName`, just in case the test wrongly passes because one of the other lines causes that exception to be thrown